engine/rpcserver.go

```diff
@@ -4578,3 +4588,151 @@ func (s *RPCServer) Shutdown(_ context.Context, _ *gctrpc.ShutdownRequest) (*gct
+		// With periods 12 fast, 26 slow, 9 periods causes a panic.
+		// var macd *kline.MACD
+		// macd, err = klines.GetMovingAverageConvergenceDivergenceOnClose(int(r.FastPeriod),
+		// 	int(r.SlowPeriod),
+		// 	int(r.Period))
+		// if err != nil {
+		// 	return nil, err
+		// }
+		// signals["MACD"] = &gctrpc.ListOfSignals{Signals: macd.MACD}
+		// signals["SIGNAL"] = &gctrpc.ListOfSignals{Signals: macd.SignalVals}
+		// signals["HISTOGRAM"] = &gctrpc.ListOfSignals{Signals: macd.Histogram}
+		return nil, common.ErrNotYetImplemented
+	case "MFI":
+		var prices []float64
+		prices, err = klines.GetMoneyFlowIndex(int(r.Period))
```

Comment:
Seems like it would be better to convert to `int` when actually calling the indicator where its required and just keep it int64 everywhere else